### ISSUE TITLE
Agenda for Feb 16, 2023

### ISSUE THREAD

**Issue by foolip** [web-platform-tests/interop#287]
Here is the proposed agenda for our meeting on Feb 16, 2023:

- Updates on action items from last week:
  - https://github.com/web-platform-tests/interop/issues/276
  - https://github.com/web-platform-tests/interop/issues/277
- https://github.com/web-platform-tests/interop/issues/256
- https://github.com/web-platform-tests/interop/issues/275
- https://github.com/web-platform-tests/interop/issues/270
  - This depends on the charter discussion
- https://github.com/web-platform-tests/interop/issues/283
  - Which changes do we want to retroactive make to 2021/2022?
  - When do we "freeze" past dashboards? See https://github.com/web-platform-tests/interop/issues/286

Apologies for posting this agenda on the same day. If we move forward with https://github.com/web-platform-tests/interop/pull/102 we'll need to get better about this, announcing at least 48 hours in advance.

Previous meeting: https://github.com/web-platform-tests/interop/issues/278

**Comment by nairnandu** [comment]
Attendees: @bkardell, @boazsender, @foolip, @jensimmons, @meyerweb, @nairnandu, @nt1m, @rnyman, @tantek, @zcorpan

*   Updates on action items from last week:
    *   [Blocking metrics updates on test regressions #276](https://github.com/web-platform-tests/interop/issues/276). Two PRs sent for review:
        *   [Write non-OK harness statuses to a log file results-analysis#150](https://github.com/web-platform-tests/results-analysis/pull/150)
        *   [Create a workflow for maintaining a list of labeled tests in-repo #288](https://github.com/web-platform-tests/interop/pull/288)
    *   [Investigation Effort goals and tracking #277](https://github.com/web-platform-tests/interop/issues/277)
        *   [zcorpan] [Issue](https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/1) filed for accessibility testing area
        *   [foolip] Was the issue mentioned within the scope of the investigation area?
        *   [boazsender] Discussing ownership for accessibility investigation area with chrishtr@, zcorpan@ and gsnedders@
*   [Scoring: variants and multi-globals #256](https://github.com/web-platform-tests/interop/issues/256)
    *   [foolip] Agreement on the ideal behavior for this issue would be a good start
    *   [gsnedders] Scoring as 1/8th would be a reasonable solution
    *   [foolip] We need to find a volunteer to do this work
    *   [gsnedders] Do we want to retroactively apply this for 2023?
    *   [jensimmons] We should stage it first so that we can review the impact
    *   [foolip] Consensus on the approach - When we have variants, we can group them using information from the manifest, score them individually, and divide by the number of variants in the group. Similarly for multi-global tests.
*   [Finish the Interop Team Charter #275](https://github.com/web-platform-tests/interop/issues/275)
    *   [foolip] This should be finished before Interop 2024
    *   [jensimmons] Let's separate them into different issues, so that we can talk about each one independently. We can aim to make these decisions before June.
    *   AI: foolip will file independent issues
    *   [foolip] Browser specific failures - on the question of having this be part of Interop scope - does anyone feel strongly about this?
    *   [jensimmons] The browser specific graph is problematic in that there is a perception that it covers all tests. Should we have more than one graph? We want to form an opinion on what should be the goal with graphs. I am good with this being discussed in this group (vs WPT group)
    *   [zcorpan] Could this graph be per directory?
    *   [gsnedders] Many things were proposed in the initial RFC. We do need to decide if this is a topic for Interop team
    *   [boazsender] I do have some thoughts on this 
    *   [foolip] What would be the desired outcome?
    *   [boazsender] Codify our perspective on what is the messaging we want to send out with WPT.
    *   [jensimmons] We could re-design the homepage to make sure it aligns with the charter
    *   [boazsender] There are stakeholders that are not part of the Interop team meeting. We should survey or get inputs from all contributors to WPT
    *   [gsnedders] Should the default be status quo? Not deleting the graph because we can’t arrive at a consensus
    *   [jensimmons] Maybe it should not be a binary decision. We definitely need to make improvements and that should be in the charter.  
    *   [bkardell] We should not leave it at status quo
    *   [jensimmons] Agree. We do have a strong appetite to put effort into the re-design
    *   [boazsender] In the very early days of WPT (at TPAC), there was concern about making wpt.fyi like an ACID test. Bringing that back to this group so that we think about that when we.
    *   [jensimmons] Data itself is misleading. Safari dropped from 5k to 3k tests because we shipped one feature (offscreen canvas). We were failing some of the tests because the test had dependencies on the offscreen canvas feature. That graph is more reflective of the way 2 engines (Chromium and Webkit) differ in the way testing (and test workflow integration) is done.
    *   [boazsender] solution that group came up with was to just show the Interop score
    *   [foolip] we can do 2 things for the cases like offscreen canvas?
        *   Weighting by feature
        *   Filter by spec maturity
    *   [jensimmons] we were failing ~1700 tests for features that Safari had shipped, but the feature tests depended on offscreen canvas, which we had not shipped at the time.
    *   [nt1m] A metric or a graph that covers the whole WPT project in general would be difficult to keep track of, in general. Filtering would be a good first step.
    *   **Consensus**: Yes to working on presenting the overall tests as part of defining the charter.
*   [Reconsider the browser specific failures graph #270](https://github.com/web-platform-tests/interop/issues/270)
    *   This depends on the charter discussion
*   [Review Interop 2022 dashboard changes #283](https://github.com/web-platform-tests/interop/issues/283)
    *   Questions:
        *   Which changes do we want to retroactively make to 2021/2022?
        *   When do we "freeze" past dashboards? See [Inconsistent end-point for Interop pages #286](https://github.com/web-platform-tests/interop/issues/286)
    *   [gsnedders] the scores above are based on the overall score, but the graph does not end at EOY
    *   [jensimmons] The data on the dashboard should be frozen at the EOY (Dec. 31st). Should be static.
    *   [tantek] I would have preferred that older dashboards retain the display that was decided for them with all their context at the time.
    *   [zcorpan] Agree to switching to a static page 
    *   [tantek] If we are freezing the scores, we should rename the header to read something other than a “dashboard”. Like “Interop 2022 conclusion” or “Interop 2022 results”. 
    *   [foolip] should we change the 2022 page to the old design?
    *   [jensimmons] agree with the idea of making any yearly changes to the dashboard be scoped to that particular year
    *   [tantek] It sets the policy moving forward. Freeze numbers and presentation of the dashboard at the end of the year.
    *   [jensimmons] If someone re-writes a test and that impacts a previous score - the old scores or static dashboard should not change
    *   [nt1m] changing the graph per focus area will be hard, if we make the change
    *   [foolip] we can freeze the data behind the graph
*   [Handling test renames or splits (test suite evolution over time) #284](https://github.com/web-platform-tests/interop/issues/284)
    *   Not discussed, but is largely addressed by freezing past year’s dashboards.
